reana/reana_dev/git.py

```diff
@@ -330,8 +330,13 @@ def git_fork(component, exclude_components, browser):  # noqa: D301
 @click.option(
     "--exclude-components", default="", help="Which components to exclude? [c1,c2,c3]",
 )
+@click.option(
+    "--target-directory", default=".", help="In which directory install? [.]"
+)
+
```

Comment:
We are using `black` Pythonic code formatter, and our CI tests are also running black-compliance checks. This is why the CI status is "red". Please plug `black` into your editor so that you could format code automatically before pushing.

Reply:
See also `flake8` that discovered a warning:

```console
$ flake8 .                                                     
./reana/reana_dev/git.py:414:9: F841 local variable 'error' is assigned to but never used
```

which makes CI to fail.  

We could screen-share about these setup-related things...

Reply:
Done